DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

Seuss27 opened the pull request:
- [ ] closes #24256

Seuss27 committed:
DOC: Closes #24256 Fix flake8 issues in whatsnew v0.13*

Seuss27 committed:
DOC: Closes #24256 Fix flake8 issues in whatsnew v0.13.*

Added the setup.cfg file

Seuss27 committed:
DOC: Fixed flake8 issues in whatsnew v0.13.* #24256

Added setup.cfg

Seuss27 committed:
DOC: Closes #24256 Fixe flake8 issues in whatsnew v0.13.*

Added the flake8-group: ignore to fix the intentional E999 errors and
fixed the remaining flake8 issues in v0.13.0.rst

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

Corrected issues in v0.13.1.rst

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

Updated setup.cfg to the most recent version before PR

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

Added missing space in setup.cfg exclusion section.

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

One more try with setup.cfg...

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

One more try with setup.cfg...

Seuss27 commented:
I am not sure if there is anything different I can do with the merge conflict in setup.cfg.  Please advise.

datapythonista commented on doc/source/whatsnew/v0.13.1.rst:
```diff
@@ -89,10 +86,13 @@ Output Formatting Enhancements
+                            pd.Timestamp('20040601')], columns=['age'])
+      >>> df['today'] = pd.Timestamp('20130419')
+      >>> df['diff'] = df['today'] - df['age']
+      >>> df
```
I'm happy with that, but I think there are `.. code-block:: text` or something like that, which would show in a block but won't raise the error.

datapythonista commented on setup.cfg:
```diff
@@ -138,7 +116,7 @@ known_post_core=pandas.tseries,pandas.io,pandas.plotting
 sections=FUTURE,STDLIB,THIRDPARTY,PRE_CORE,DTYPES,FIRSTPARTY,POST_CORE,LOCALFOLDER
 
 known_first_party=pandas
-known_third_party=Cython,numpy,python-dateutil,pytz,pyarrow,pytest
```
can you undo this change here, and send it in a different PR? I don't think this is related to the rest of the changes, is it?

Seuss27 replied:
setup.cfg was rolled back completely and corrected.

datapythonista commented on setup.cfg:
```diff
@@ -45,19 +45,6 @@ ignore = E402,  # module level import not at top of file
-    doc/source/whatsnew/v0.13.0.rst
-    doc/source/whatsnew/v0.13.1.rst
-    doc/source/whatsnew/v0.14.0.rst
-    doc/source/whatsnew/v0.14.1.rst
```
I'm a bit confused about this. Are all the others correct and we forgot to remove them? I'd expect just the `v0.13.*` to be removed

saurav2608 replied:
Sorry if I am butting in. Issues related to v0.14* was fixed and merged. I think a merge with upstream/master will resolve this.

Seuss27 replied:
Thanks for the feedback on this, I hadn't had to resolve a merge conflict before and I was doing it wrong.  All sorted now I believe.

datapythonista reviewed (commented):
Good changes, just added a few comments. Also, you'll have to fix the conflicts, I guess we removed some more fixed pages from `setup.cfg`

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

Another attempt to get the conflicts resolved with setup.cfg
Incorporated the requested changes.

Seuss27 committed:
Merge branch 'v0.13' of https://github.com/Seuss27/pandas into v0.13

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

Rollback strange changes to setup.cfg and hopefully resolve
the merge conflict.

Seuss27 committed:
Revert "DOC: Closes #24256 Fix flake8 issues in whatsnew v0.13*"

This reverts commit bb06bdb63f141acd85dd83bf57dd7df1dbbec1b9.

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

Another attempt to fix the merge conflict with setup.cfg

datapythonista commented on doc/source/whatsnew/v0.13.0.rst:
```diff
@@ -123,21 +119,30 @@ API changes
+     a.bool(), a.item(), a.any() or a.all().
+
+     >>> df1 and df2
+     ValueError: The truth value of a DataFrame is ambiguous.  Use a.empty,
```
did you see if this generates a warning when building the docs? we may need to use this syntax if that's the case: https://docs.python.org/3/library/doctest.html (see `Traceback`...)

Seuss27 replied:
This throws an indention error and then an undefined error when the df1 and df2 gets indented (to aling with the if df: above I think.  Since df1, df2, s1, and s2 are never defined I wasn't sure how to proceed.  Should I define them or follow the traceback approach?

Seuss27 replied:
I ended up assigning the variables, let me know if you'd like to see an alternative solution...

datapythonista reviewed (commented):
Can you do the merge from master, and fix the conflicts please. We can't merge this PR otherwise.

jreback commented:
pls merge master

Seuss27 committed:
Merge branch 'master' into v0.13

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

Fixed merge errors with setup.cfg, reapplying changes in this version

Seuss27 committed:
DOC: WIP v13.0 whatsnew

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.* #24256

Implemented remaining requested changes...

Seuss27 committed:
DOC: Fixes flake8 issues in whatsnew v0.13.*

jreback commented:
thanks @Seuss27
